src/components/DropDown.tsx

```diff
@@ -0,0 +1,140 @@
+import React, {useEffect, useRef} from "react";
+import styled from "styled-components";
+import {DropdownProps} from "../types";
+import Icon from "@mdi/react";
+import {mdiContentCopy} from "@mdi/js";
+import {CoinLogos, copyText, getAbbreviatedAddress, getErrorMessage} from "../utils";
+import ReactLoading from "react-loading";
+import {baseStyle} from "../styles/baseStyle";
+
+const DropDown = (props: DropdownProps) => {
+  const {active, content, loading, error, onChange, onClickOutside} = props;
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const handleClickOutside = (event) => {
+      if (ref.current && !ref.current.contains(event.target)) {
+        onClickOutside && onClickOutside();
+      }
+    };
+    document.addEventListener('click', handleClickOutside, true);
+    return () => {
+      document.removeEventListener('click', handleClickOutside, true);
+    };
+  }, [onClickOutside]);
+
+  return active ? (
+    <DropDownWrapper ref={ref}>
+      {loading ? <StyledLoader type="spinningBubbles" color={baseStyle.primaryColor} height={baseStyle.loader.height}
+                               width={baseStyle.loader.height}/> : null}
+      {!loading && content.length > 0 ? <UnorderedList>
+        {content.sort((a, b) => a.network.localeCompare(b.network)).map((item, key) => {
+          return (
+            <ListItem key={key}>
+              <ItemWrapper>
+                <StyledContent>
+                  <StyledLogo width={baseStyle.dropDown.logo.width} src={CoinLogos[item.network.toLocaleUpperCase()]}
+                              alt="coinLogo"/>
```

Comment:
Logo not clickable?
Let`s do the full field clickable but prevent the mdiContentCopy click

Reply:
I don't see the point in that

Reply:
Okay, we will look on it tomorrow
May be it`s a problem only for me)